Saucedemo/Core/Wrappers/ItemsList.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OpenQA.Selenium;
+
+namespace Saucedemo.Core.Wrappers
+{
+    public class ItemsList
+    {
+        private readonly UiElement _uiElement;
+        private readonly List<ListItem> _listItems = new List<ListItem>();
+
+        public ItemsList(IWebDriver driver, By by, string itemClassName)
+        {
+            _uiElement = new UiElement(driver, by);
+
+            foreach (var webElement in _uiElement.FindElements(By.ClassName($"{itemClassName}")))
```

Comment:
thats implementation calls `_uiElement.FindElements` in each iteration. Does it make sense?